src/Fixture/AbstractResourceFixture.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Fixture;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Bundle\FixturesBundle\Fixture\Factory\ExampleFactoryInterface;
+use Sylius\Bundle\FixturesBundle\Fixture\FixtureInterface;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+abstract class AbstractResourceFixture implements FixtureInterface
```

Comment:
Do we even need to add `Resource` here? If I'm not missing something, this class could be used in any context, so just the creation of a sample structure for fixture and executing it in PHPunit would be enough(still not a no-brainer). What do you think, Loic?

Reply:
You're right, we don't need Resource suffix. I will try a phpunit test tomorrow.. is there any similar tests on this bundle ?

Reply:
Not yet. These are closest I suppose: https://github.com/Sylius/SyliusFixturesBundle/tree/master/tests/Loader

Reply:
@lchrusciel AbstractFixture already exists and it's a minimalist version without any example factory.

Reply:
What do you think about renaming it to `AbstractExampleFactoryAwareFixture`?

Reply:
@lchrusciel I renamed it, I think this is a better name.
I don't know when I will add some phpUnit tests yet.

Reply:
Don't worry. Maybe someone will pick it up

Reply:
@lchrusciel Cause it's a little too long, I'm afraid developers doesn't remember which abstract service he should extend.
What do you think about `AbstractConfigurableFixture`?